hack/managed-kyma-migration/migration-testing/migrate-local-moduletemplate.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Since we are using relative paths in this script we need to change directory to the script's to not depend on current working directory during the execution
+script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$script_path"
+
+# Fetch Kyma CR name managed by lifecycle-manager
+kyma_cr_name=$(kubectl get kyma -n kyma-system --no-headers -o custom-columns=":metadata.name")
+
+# Check if Istio is already present on Kyma CR
```

Comment:
I think we can remove the check for multiple Istio CRs, since it's not used anywhere other than for the echo output.

Reply:
In lines 13-18, this is a check for Istio entries in KymaCR. We need it to exit early in case when it is already there.